src/api/route-services/samples.js

```diff
@@ -50,7 +54,36 @@ class SamplesService {
       return prettyResponse;
     }
 
-    throw Error('Sample not found');
+    throw new NotFoundError('Samples not found');
+  }
+
+  async updateSamples(projectUuid, body) {
+    const marshalledKey = convertToDynamoDbRecord({
+      experimentId: body.experimentId,
+    });
+
+    const marshalledData = convertToDynamoDbRecord({
+      ':samples': body.samples,
+      ':projectUuid': projectUuid,
+    });
+
+    // Update samples
```

Comment:
do we have samples copied across different tables? Because if we do, then we will need to update all places where we store samples